dev/devicelab/bin/run.dart

```diff
@@ -95,10 +100,16 @@ Future<void> main(List<String> rawArgs) async {
       stderr.writeln(argParser.usage);
       exit(1);
     }
+    if (localEngineHost == null) {
+      stderr.writeln('When running in A/B test mode --local-engine-host is required.\n');
```

Comment:
Are we going to be hitting this when we land this?

Reply:
My understanding is A/B test mode is just run locally (not on CI), so this should be fine with instructions?

(I could make it optional, but it potentially makes the code much more complex)

Reply:
Okay, I think this is good.